src/globus_action_provider_tools/authentication.py

```diff
@@ -218,39 +217,34 @@ def get_dependent_tokens(
     def get_authorizer_for_scope(
         self,
         scope: str,
-        required_authorizer_expiration_time: int = 60,
-    ) -> RefreshTokenAuthorizer | AccessTokenAuthorizer:
+        required_authorizer_expiration_time: int | None = None,
+    ) -> AccessTokenAuthorizer:
         """
         Get dependent tokens for the caller's token, then retrieve token data for the
         requested scope and attempt to build an authorizer from that data.
 
         The class caches dependent token results, regardless of whether or not
         building authorizers succeeds.
 
-        .. warning::
-
-            This call converts *all* errors to `None` results.
-
-        If a dependent refresh token is available in the response, a
-        RefreshTokenAuthorizer will be built and returned.
-        Otherwise, this will attempt to build an AccessTokenAuthorizer.
-
-        If the access token is or will be expired within the
-        ``required_authorizer_expiration_time``, it is treated as a failure.
+        :param scope: The scope for which an authorizer is being requested
+        :param required_authorizer_expiration_time: Deprecated parameter. Has no effect.
         """
+        if required_authorizer_expiration_time is not None:
+            warnings.warn(
+                "`required_authorizer_expiration_time` has no effect and will be removed in a future version.",
+                DeprecationWarning,
+                stacklevel=1,
```

Comment:
I think this should be at least level 2 so people can see where in their code they've called this.

```suggestion
                stacklevel=2,
```

Reply:
Ah, yes, good catch!

I find this parameter hard to index correctly in my head because in some sense it counts from 1 rather than 0.